Ambit/Source/AWSUE4Module/Private/FirehoseUE4Client.cpp

```diff
@@ -65,22 +64,22 @@ void FirehoseUE4Client::SendEvents(FString StreamName, TArray<FString> Data) con
     {
         Aws::Firehose::Model::Record Record;
 
-        Aws::Utils::ByteBuffer bytes((unsigned char*)TCHAR_TO_UTF8(*Element), Element.Len());
+        Aws::Utils::ByteBuffer bytes(reinterpret_cast<unsigned char*>(TCHAR_TO_UTF8(*Element)), Element.Len());
 
         Record.SetData(bytes);
         Records.emplace_back(Record);
     }
 
     Request.SetRecords(Records);
 
-    Client->PutRecordBatchAsync(Request, [](const Aws::Firehose::FirehoseClient*,
-                                            const Aws::Firehose::Model::PutRecordBatchRequest&,
+    Client->PutRecordBatchAsync(Request, [](const Aws::Firehose::FirehoseClient* /*unused*/,
```

Comment:
Normally definitions of functions (even lambdas) have names attached to them. This lambda foregoes that for some of the parameters, but ReSharper knows that they should be _something_, it just doesn't know what, and knows that they can't be utilized since it isn't referenced in the lambda code. So, it puts the "unused" comment here to represent that there should probably be parameter names, but won't give them a value.